Makefile

```
@@ -20,9 +24,17 @@ local: pip_update  ## Install local requirements and dependencies
 outdated:  ## Check outdated requirements and dependencies
 	python3 -m pip list --outdated
 
+.PHONY: permissions
+permissions:  ## Generate permissions csv file
+	python3 manage.py permissions --path artifacts/permissions.csv
+
 .PHONY: pip
 pip: pip_update  ## Compile requirements
-	python3 -m piptools compile --no-header --quiet --upgrade --output-file requirements.txt requirements.in
+	python3 -m piptools compile --no-header --quiet --upgrade --output-file requirements/base.txt requirements/base.in
+	python3 -m piptools compile --no-header --quiet --upgrade --output-file requirements/common.txt requirements/common.in
+	python3 -m piptools compile --no-header --quiet --upgrade --output-file requirements/local.txt requirements/local.in
+	python3 -m piptools compile --no-header --quiet --upgrade --output-file requirements/remote.txt requirements/remote.in
+	python3 -m piptools compile --no-header --quiet --upgrade --output-file requirements/test.txt requirements/test.in
```

```suggestion
	python3 -m piptools compile --no-header --quiet --upgrade --output-file requirements/common.txt requirements/common.in
	python3 -m piptools compile --no-header --quiet --upgrade --output-file requirements/local.txt requirements/local.in
	python3 -m piptools compile --no-header --quiet --upgrade --output-file requirements/test.txt requirements/test.in
```
We agreed with @pauloxnet to only user 3 levels.